src/status.rs

```diff
@@ -431,15 +455,31 @@ impl StatusBuilder {
         self
     }
 
+    /// Add an extension to the `Status` object.
+    ///
+    /// The prefix `ext_` will automatically be prepended to the name, if not
+    /// already present.
+    pub fn add_extension<S: Into<String>>(mut self, name: S, value: Value) -> Self {
+        let mut key = name.into();
+        if !key.starts_with("ext") {
+            key = String::from("ext_") + &key;
```

Comment:
Same as previous, check for `starts_with("ext_")` otherwise it will fail with something like "extrem_stuff".